vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -22,15 +26,23 @@ packages:
         - libpolkit-agent-1-0
         - libpolkit-gobject-1-0
         - apache2-utils
+        - apt-transport-https
+        - ca-certificates
+        - gnupg-agent
+        - software-properties-common
```

Comment:
these packages are not needed when `CONTAINER_RUNTIME=podman`

Reply:
Fixed.